src/main/java/frc/robot/components/TrigonXboxController.java

```diff
@@ -21,86 +21,68 @@
-        rightBumper = new JoystickButton(this, Button.kBumperRight.value);
-        leftStickButton = new JoystickButton(this, Button.kStickLeft.value);
-        rightStickButton = new JoystickButton(this, Button.kStickRight.value);
+        a = new JoystickButton(this, Button.kA.value);
+        b = new JoystickButton(this, Button.kB.value);
+        x = new JoystickButton(this, Button.kX.value);
+        y = new JoystickButton(this, Button.kY.value);
+        leftBumper = new JoystickButton(this, Button.kLeftBumper.value);
+        rightBumper = new JoystickButton(this, Button.kRightBumper.value);
+        leftStickButton = new JoystickButton(this, Button.kLeftStick.value);
+        rightStickButton = new JoystickButton(this, Button.kRightStick.value);
 
         backButton = new JoystickButton(this, Button.kBack.value);
         startButton = new JoystickButton(this, Button.kStart.value);
         rumbleAmount = -1;
         notifier = new Notifier(this::notifierPeriodic);
     }
 
-    public JoystickButton getButtonA() {
-        return buttonA;
+    public JoystickButton getABtn() {
+        return a;
     }
 
-    public JoystickButton getButtonB() {
-        return buttonB;
+    public JoystickButton getBBtn() {
+        return b;
     }
 
-    public JoystickButton getButtonX() {
-        return buttonX;
+    public JoystickButton getXBtn() {
+        return x;
     }
 
-    public JoystickButton getButtonY() {
-        return buttonY;
+    public JoystickButton getYBtn() {
+        return y;
     }
 
-    public JoystickButton getLeftBumper() {
+    public JoystickButton getLeftBumperBtn() {
         return leftBumper;
     }
 
-    public JoystickButton getRightBumper() {
+    public JoystickButton getRightBumperBtn() {
         return rightBumper;
     }
 
-    public JoystickButton getLeftStickButton() {
+    public JoystickButton getLeftStickButtonBtn() {
         return leftStickButton;
     }
 
-    public JoystickButton getRightStickButton() {
+    public JoystickButton getRightStickButtonBtn() {
         return rightStickButton;
     }
 
-    public JoystickButton getBackXboxButton() {
+    public JoystickButton getBackBtn() {
         return backButton;
     }
 
-    public JoystickButton getStartXboxButton() {
+    public JoystickButton getStartBtn() {
         return startButton;
     }
 
-    /**
-     * This method returns the positive trigger minus the negative trigger. This is
-     * used the for controlling the speed on the Y axis of the robot.
-     *
-     * @param positiveHand The hand that is positive, the other will be negative.
-     * @return positive trigger value - other trigger value.
-     */
-    public double getDeltaTriggers(Hand positiveHand) {
-
-        if(positiveHand == Hand.kRight)
-            return getTriggerAxis(Hand.kRight) - getTriggerAxis(Hand.kLeft);
-        else
-            return getTriggerAxis(Hand.kLeft) - getTriggerAxis(Hand.kRight);
-    }
```

Comment:
keep just change the logic to:
`return positiveHand == Hand.kRight ? getDeltaTriggers() : -getDeltaTriggers()`

Reply:
There's no `Hand` class anymore

Reply:
than find a boolean instead

Reply:
boolean? that's ugly. Just use the default. worst case just multiply this by -1.